kafka-rest/src/main/java/io/confluent/kafkarest/resources/v3/ProducerMetrics.java

```diff
@@ -111,8 +111,14 @@ final class ProducerMetrics {
   private final String responseSensorName;
   private final String recordErrorSensorName;
   private final String requestLatencySensorName;
+  private final Map<String, String> metricsTags;
 
   ProducerMetrics(KafkaRestConfig config, Time time) {
+    this(config, time, ImmutableMap.of());
+  }
+
+  ProducerMetrics(KafkaRestConfig config, Time time, Map<String, String> metricsTags) {
```

Comment:
If all the difference between this and the one we bind in `ce-kafka-rest` is the `metricsTags`, we should just bind a different `metricsTags` instead there, and inject it here. Food for thought.

Reply:
Never mind. Let's default to bind less things instead of more. It's not like HK2 is super nice anyway.